### PR TITLE
Testing/nl book info

### LINKED ISSUE #15: Test Book Info component (initial test)

### PR DESCRIPTION
**Pull request by NicoleLatifi**
#### What's this PR do?
- Add test for Book Info component that tests for the correct content to render.

- Also made some styling updates to the Book Info page to make it match the header styling and be responsive.

Type of change made:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features
- [x] Testing

#### Where should the reviewer start?
- src/Components/BookInfo/BookInfo.test.js

#### Why is this change required? What problem does it solve?
- Test Book Info component.

#### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- n/a

#### Any background context you want to provide?
-  n/a

#### What are the relevant tickets?
- resolves #15 

#### Screenshots (if appropriate)
Screen Shot 2020-09-11 at 1 33 13 PM

Screen Shot 2020-09-11 at 1 34 07 PM

#### Questions:
- n/a

### CODE REVIEW

**Review by amykarnaze — COMMENTED** [review]
Nice job getting into testing.

File: src/Components/BookInfo/BookInfo.css [hunk 1]
```diff
@@ -13,24 +15,72 @@
...
+  text-decoration: none;
   width: 100px;
 }
+
+.add-to-reading-list:hover, .purchase:hover {
+  background:  #F4DBD8;
+  transition: 0.5s;
+}
+
+@media only screen and (max-width: 800px) {
```
Comment (kip-west) [inline src/Components/BookInfo/BookInfo.css:60]: This styling looks really great!! And great thought to start making our website responsive!!

**Review by kip-west — APPROVED** [review]
This looks great!! Great job getting unit tests working for this display & getting the test suite going.

File: src/Components/BookInfo/BookInfo.css (hunk 1 above)
Comment (NicoleLatifi) [reply src/Components/BookInfo/BookInfo.css:60]: Thanks! I can do similar to the other components if it seems like a breakpoint we want to go with.

**Comment by NakiNorton** [comment]
Looking great! Love the new font on the buttons. Pulls everything together nicely.

**Pull request merged**